app/views/shared/_date_of_birth_form_field.html.erb

```diff
@@ -10,49 +9,49 @@
     <label for="<%= form_element_day_id %>">
       <span class="sr-only">Date of birth</span> <%= required_label(:day) %>
       <%=
-        form.number_field(
+        form.text_field(
           'date_of_birth(3i)',
           id: form_element_day_id,
           use_label: false,
           value: form.object.date_of_birth.try(:day),
           placeholder: 'DD',
           class: 'form-dob__input js-dob-day t-date-of-birth-day',
-          pattern: '[0-9]*',
-          min: 1,
-          max: 31,
+          pattern: '(0?[1-9]|(1|2)[0-9]|3[0-1])',
+          maxlength: 2,
+          required: true,
           'aria-describedby': 'dob-hint'
```

Comment:
If we add a `title` attribute to these fields, it'll be displayed as part of the error.

So perhaps `title="Must be numeric"` or something?

<img width="386" alt="screen shot 2016-11-22 at 14 00 20" src="https://cloud.githubusercontent.com/assets/295469/20526445/02751ed8-b0bc-11e6-99c9-06ae85875a88.png">